app/models/glider.rb

```diff
@@ -13,7 +13,11 @@ class Glider < ApplicationRecord
   has_many :attendances
 
   def name
-    "#{glider_type} #{size} | #{manufacturer}"
+    if manufacturer.nil?
```

Comment:
If `manufacturer` can also be empty string (I assume it could be), then the safer variant for this if statement would be `if manufacturer.blank?`. `blank?` captures both `nil` and empty string.

Reply:
cool, I'll change that